package.json

```diff
@@ -10,7 +10,7 @@
   "name": "fastly",
   "description": "Fastly API client for Node.js",
   "keywords": "fastly, api, client",
-  "version": "2.1.0",
+  "version": "2.5.0",
```

Comment:
not sure if you want to do a major version bump if you are changing some of the function names